scripts/prepare_sector_network.py

```diff
@@ -2390,7 +2405,7 @@ def add_biomass(n, costs):
     n.madd(
         "Bus",
         spatial.gas.biogas,
-        location=spatial.gas.locations,
+        location=spatial.gas.demand_locations,
```

Comment:
Strictly speaking the biogas is not a demand but a potential. I am a bit unhappy that an option called  `regional_gas_demand` , also regionalizes biogas production.  I think it would be cleaner to have two options: `regional_gas_demand` AND `regional_biogas`. Biogas should then be assigned to `spatial.gas.biogas_locations`